core/client_test.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 var (
-	fixtureInvalidAPITokenErr = "invalid_api_token: The API token provided " +
-		"was not valid (it may not exist or have expired)"
+	fixtureInvalidAPITokenErr = "katapult: unauthorized: invalid_api_token: " +
+		"The API token provided was not valid " +
+		"(it may not exist or have expired)"
```

Comment:
Is this using string concat because of `lll` ?

Reply:
Yeah, but also cause I personally really dislike long lines :P